e2e/various-suite/explore.spec.ts

```diff
@@ -46,7 +46,7 @@ e2e.scenario({
 
     // Both queries above should have been run and be shown in the query history
     e2e.components.QueryTab.queryHistoryButton().should('be.visible').click();
-    e2e.components.QueryHistory.queryText().should('have.length', 2).should('contain', 'csv_metric_values');
+    e2e.components.QueryHistory.queryText().should('have.length', 4).should('contain', 'csv_metric_values');
```

Comment:
I dont like this, could be different based on different default values, will make more robust